bosh-director/spec/unit/deployment_plan/stemcell_spec.rb

```diff
@@ -9,8 +9,8 @@ def make_deployment(name)
     BD::Models::Deployment.make(:name => name)
   end
 
-  def make_stemcell(name, version, os = 'os1')
-    BD::Models::Stemcell.make(:name => name, :operating_system=>os, :version => version)
+  def make_stemcell(name, version, os = 'os1', params = {})
```

Comment:
It's not only `cpi`, but also `cid` (see further below in the spec). Should I add both as separate arguments? I prefer the `params` way since it's more readable.